lang/driver/src/paths.rs

```diff
@@ -3,3 +3,5 @@ pub const TARGET_PATH: &str = "target_pol/";
 pub const DOCS_PATH: &str = "target_pol/docs/";
 
 pub const CSS_PATH: &str = "target_pol/docs/style.css";
+
+pub const CSS_TEMPLATE_PATH: &str = "lang/docs/templates/style.css";
```

Comment:
This only works when executing the tool from the polarity directory. We probably want to include this file in the static part of the executable. This is possible using the [`include_str!`](https://doc.rust-lang.org/std/macro.include_str.html) macro.
You can then write it to the `target/docs` directory, e.g. using [`fs::write`](https://doc.rust-lang.org/beta/std/fs/fn.write.html).